src/Razor/src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/ProjectSystem/MacRazorProjectHostBase.cs

```diff
@@ -204,24 +203,17 @@ protected void RemoveDocument(HostProject hostProject, string filePath)
         }
     }
 
-    // Internal for testing
-    internal static bool TryGetIntermediateOutputPath(
+    protected bool TryGetIntermediateOutputPath(
         IMSBuildEvaluatedPropertyCollection projectProperties,
         [NotNullWhen(returnValue: true)] out string? path)
     {
-        if (!projectProperties.HasProperty(BaseIntermediateOutputPathPropertyName))
-        {
-            path = null;
-            return false;
-        }
-
         if (!projectProperties.HasProperty(IntermediateOutputPathPropertyName))
         {
             path = null;
             return false;
         }
 
-        var baseIntermediateOutputPathValue = projectProperties.GetValue(BaseIntermediateOutputPathPropertyName);
+        var baseIntermediateOutputPathValue = DotNetProject.BaseIntermediateOutputPath.ToString();
```

Comment:
This fixes the issue, because VS Mac will give us the fully qualified path from this property, so everything works

Reply:
So `projectProperties.GetValue(BaseIntermediateOutputPathValue)` is a relative path then?